cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html

```diff
@@ -0,0 +1,119 @@
+<section class="college-costs_tool-section costs" data-tool-section="costs">
```

Comment:
Should `costs` be a less ambiguous classname? Like `college-costs_tool-costs` or something?

Reply:
That's a good point. Updated to `college-costs_tool-section__costs` in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c